composer.json

```diff
@@ -17,16 +17,17 @@
     "extra": {
     },
     "require": {
-        "php": "^7.1",
+        "php": "^7.3 || ~8.0.0",
         "laminas/laminas-config-aggregator": "^1.1",
         "laminas/laminas-stdlib": "^3.1",
         "laminas/laminas-zendframework-bridge": "^1.0",
         "symfony/dependency-injection": "^3.4.36 || ^4.4.2 || ^5.0"
     },
     "require-dev": {
+        "dms/phpunit-arraysubset-asserts": "^0.2.1",
```

Comment:
Maybe not the best solution but I did not want to change tests.

Reply:
Makes perfect sense to me :+1: